main.py

```diff
@@ -122,6 +130,8 @@ def getcreds(self):
                 click.echo(click.style("RAISED > ", fg="green", bold=True) + str(len(purltext)) + " result(s) in " + str(duration)[0:3] + " seconds [" + self.obtntime() + "]")
             except wikipedia.exceptions.HTTPTimeoutError:
                 click.echo(click.style("CAVEAT > ", fg="red", bold=True) + "Timeout occured while processing the query.")
+            except wikipedia.exceptions.PageError:
+                click.echo(click.style("CAVEAT > ", fg="red", bold=True) + "Couldn't find the wikipedia page for the query.")
```

Comment:
As Wikipedia is a proper noun, you are requested to change `wikipedia` to `Wikipedia`.